libcore/src/Simulation.cpp

```diff
@@ -556,16 +550,16 @@ double Simulation::RunBody(double maxSimTime)
                 int id           = atoi(strs[1].c_str());
                 std::string type = Trans->GetCaption();
                 trainOutflow[id] += Trans->GetDoorUsage();
-                if(trainOutflow[id] >= TrainTypes[type]->nmax) {
+                if(trainOutflow[id] >= _building->GetTrainTypes().at(type).nmax) {
```

Comment:
We should catch the exception... Maybe outside this function and Log the Error.

Reply:
added try catch around RunStandardSimulation in main.cpp